lib/file-helper.unit.test.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-env jest */
+
+const fileHelper = require('../lib/file-helper')
+
+describe('getComponentData', () => {
```

Comment:
I like the idea of these tests, but it feels like they're mostly testing the contents of the YAML files rather than the `getComponentData` function itself.

Potentially they also overlap quite a lot with the package tests?

https://github.com/alphagov/govuk-frontend/blob/1abe3e3efb38aad972cc0644387b130dbea8f86f/tasks/gulp/__tests__/after-build-package.test.js#L133-L157

If we think they're useful, I wonder if we need to tweak the descriptions or something to make this clearer?